incubator/hydra/values.yaml

```diff
@@ -0,0 +1,103 @@
+image: "oryd/hydra"
+imageTag: "v0.8.6"
+imagePullPolicy: "Always"
```

Comment:
If I specify a version like `v0.9.1` I would not expect it to change. If this is actually the case, then `Always` should be a conscious choice. I'd suggest you change the default to `IfNotPresent`.